libs/designsystem/item/src/item.component.ts

```diff
@@ -57,9 +70,18 @@ export class ItemComponent {
     }
   }
 
+  /** Focus the internal button of the ion-item */
+  focusItem(item: HTMLIonItemElement) {
+    const root = item.shadowRoot;
+    const button = root.querySelector('button');
+    if (button) {
+      button.focus();
+    }
+  }
+
   get _isIonicButton() {
     // Ionic checks for slotted checkbox and radio
     // and we shouldn't set the `button` prop in that scenario:
-    return this.selectable && !(this.checkbox || this.radio);
+    return this.selectable; // && !(this.checkbox || this.radio);
```

Comment:
Even though I remove this checkbox check, I still can't get the checkbox to be checked when its within an item within a menu (see the advanced menu example where the toggle may be toggled while the checkbox may not)